ui/src/app/pages/workspace/featured-workspaces.tsx

```diff
@@ -0,0 +1,302 @@
+import * as React from 'react';
```

Comment:
Basically copied from workspace-library with minor tweaks and changing the source to getFeaturedWorkspacesByCategory.